apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentationModal/_components/APIUsage.tsx

```diff
@@ -0,0 +1,48 @@
+import { CodeBlock, Text } from '@latitude-data/web-ui'
+
+export function APIUsage({
+  projectId,
+  commitUuid,
+  documentPath,
+  apiKey,
+  parameters,
+}: {
+  projectId: number
+  commitUuid: string
+  documentPath: string
+  apiKey: string | undefined
+  parameters: Set<string>
+}) {
+  const getRequestBodyContent = () => {
+    const bodyContent = [`    "path": "${documentPath}"`]
+
+    if (parameters.size > 0) {
+      const parameterEntries = Array.from(parameters)
+        .map((key) => `      "${key}": ""`)
+        .join(',\n')
+
+      bodyContent.push(`    "parameters": {
+${parameterEntries}
+    }`)
+    }
+
+    return bodyContent.join(',\n')
+  }
```

Comment:
Weird hardcodded spacing in the strings

Reply:
I get that's the indentation for the body, but could you just add the indentation afterwards in a function so that it's simpler? Or just add it to an object and stringify later?